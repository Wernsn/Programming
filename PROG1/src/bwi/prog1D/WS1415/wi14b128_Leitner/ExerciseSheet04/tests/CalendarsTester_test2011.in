20
2004 02 29
