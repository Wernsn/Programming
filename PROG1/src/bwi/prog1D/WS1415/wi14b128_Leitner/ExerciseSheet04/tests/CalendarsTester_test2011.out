statistics for
Sunday, February 29th 2004
2004 is a leap year
February 2004 has 29 days
it is day 60 of year 2004
it is in week 9 of year 2004
